build.sbt

```diff
@@ -0,0 +1,57 @@
+lazy val scalacheck = (project in file("."))
+.settings(publishSettings:_*)
+.enablePlugins(ExerciseCompilerPlugin)
+.settings(
+  organization := "org.scala-exercises",
+  name         := "exercises-scalacheck",
+  scalaVersion := "2.11.7",
+  version := "0.1.1",
+  resolvers ++= Seq(
+    Resolver.sonatypeRepo("snapshots"),
+    Resolver.sonatypeRepo("releases")
+  ),
+  libraryDependencies ++= Seq(
+    "org.scalatest" %% "scalatest" % "2.2.4",
+    "com.fortysevendeg" %% "github4s" % "0.5",
```

Comment:
This is probably not need here anymore since @dialelo just merged a fix for this